src/org/openj9/envInfo/EnvDetector.java

```diff
@@ -69,21 +73,14 @@ private static void getJavaInfo() {
 		} catch (IOException e) {
 			e.printStackTrace();
 		}
-
 	}
 
 	private static void getMachineInfo() {
 		MachineInfo machineInfo = new MachineInfo();
-
-		machineInfo.getMachineInfo(MachineInfo.UNAME_CMD);
-		machineInfo.getMachineInfo(MachineInfo.SYS_ARCH_CMD);
-		machineInfo.getMachineInfo(MachineInfo.PROC_ARCH_CMD);
-		machineInfo.getMachineInfo(MachineInfo.SYS_OS_CMD);
-		machineInfo.getMachineInfo(MachineInfo.CPU_CORES_CMD);
-		machineInfo.getMachineInfo(MachineInfo.ULIMIT_CMD);
-
-		machineInfo.getRuntimeInfo();
-		machineInfo.getSpaceInfo("");
+		machineInfo.getInfo();
+		System.out.println("MACHINE INFO");
+		System.out.println("**************************************************************************");
```

Comment:
Could we have `MACHINE INFO` and `*` in one line? It will be easier to parse later. Something like: `************************* MACHINE INFO *********************************`

Reply:
updated

Reply:
thanks